explorer/src/queries/chainlinkNodes.ts

```diff
@@ -0,0 +1,37 @@
+import { Connection, SelectQueryBuilder } from 'typeorm'
+import { ChainlinkNode } from '../entity/ChainlinkNode'
+import { PaginationParams } from './pagination'
+
+const queryBuilder = (
```

Comment:
We should be using a custom repository for this I think. Take a look at `JobRunRepository`